CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ CHANGELOG
 * The `DatabaseReader` methods `city()` and `country()` can now be called on
   the Enterprise database and the `country()` method can be called on City
   databases. Request by Gergely Boromissza. GitHub #132.
+* New `getStaticIpScore()` and `getUserCount()` methods were added to
+  `com.maxmind.geoip2.record.Traits` for use with GeoIP2 Precision Insights.
```

Comment:
Should we say anything about what they are? I think in the other libraries we've been doing that.